xgboost_ray/examples/simple_tune.py

```diff
@@ -38,6 +39,13 @@ def train_breast_cancer(config, ray_params):
         evals_result["eval"]["error"][-1]))
 
 
+def load_best_model(checkpoint_dir):
+    """Loads the XGBoost model."""
+    best_bst = xgb.Booster()
+    best_bst.load_model(os.path.join(checkpoint_dir, "tuned.xgb"))
+    return best_bst
+
+
```

Comment:
Can we consolidate this with the client check and put into a proper xgboost_ray util?

Reply:
Done!